src/components/implementation/tests/micro_booter/micro_booter.c

```diff
@@ -568,7 +568,7 @@ term_fn(void *d)
 void
 cos_init(void)
 {
-	cos_meminfo_init(&booter_info.mi, BOOT_MEM_KM_BASE, COS_MEM_KERN_PA_SZ);
+	cos_meminfo_init(&booter_info.local_mi, BOOT_MEM_KM_BASE, COS_MEM_KERN_PA_SZ, BOOT_CAPTBL_SELF_LOCAL_PT);
```

Comment:
I thought we discussed making local default, and "global" the one that causes changes of variable names.  Local is the one that is typical.  Global is the one that is aberrant.  Why would we change all of the code to make local the one that (from a variable name-space perspective) is special cased?